renewer/tasks/renewals.py

```diff
@@ -0,0 +1,74 @@
+from huey import crontab
+
+from renewer.models.cdn import CdnRoute
+from renewer.db import SessionHandler
+from renewer.models.domain import DomainRoute
+from renewer.huey import huey
+from renewer.tasks import alb, cdn, iam, letsencrypt, s3
+
+
+@huey.periodic_task(crontab(month="*", day="*", hour="12", minute="0"))
+def renew_all_certs():
+    routes = []
+    with SessionHandler() as session:
+        for Route in (CdnRoute, DomainRoute):
+            routes.extend(Route.find_active_instances(session))
```

Comment:
The refactoring makes even more sense seeing this - for a moment I thought you were referencing a model that wasn't imported (`Route`), but then I saw it's within the `for ... in` statement, so I see what you're doing.  Nicely done!